src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_ldm3d.py

```diff
@@ -0,0 +1,699 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2023 LDM3D Authors and The HuggingFace Team. All rights reserved.
```

Reply:
Done

Reply:
I added the copyright header to the pipeline as well. Should I do the same for the image_processor?